src/App.tsx

```diff
@@ -1,14 +1,30 @@
 import { AppHeader } from "./components/AppHeader";
 import { AppLayout } from "./components/AppLayout";
-import {Card} from "./components/Card";
+import {NewCard} from "./components/NewCard";
+import React, { useState } from "react";
 
 import "./App.css";
-import React from "react";
 
 function App() {
+  const [editMode, setEditMode] = useState<boolean>(true);
+  const [cardList, setCardList] = useState<JSX.Element[]>([]); 
+
+  const changeMode = () => {
+    setEditMode(true);
+  };
+
+  const addCard = () => {
+    const newCard = <NewCard editMode={editMode} setEditMode={setEditMode} />;
```

Comment:
**issue:** Adding new card should set data object instead of JSX element. However it was not in this task scope, so you can pass this for now